apps/expo/babel.config.js

```diff
@@ -9,7 +31,12 @@ module.exports = function (api) {
   return {
     presets: ["babel-preset-expo"],
     plugins: [
-      "nativewind/babel",
+      [
+        "nativewind/babel",
+        {
+          tailwindConfig: lazyTranspileTailwindConfig(),
```

Comment:
Can we use the `loadConfig` exported from `tailwindcss/loadConfig`? It uses jiti and sucrase too.

Would feel more "official"

Reply:
Good idea, fewer dependencies as well :)

PR is updated with `loadConfig` from Tailwind.